NewHorizons/Builder/Props/TranslatorText/TranslatorTextBuilder.cs

```diff
@@ -75,62 +75,64 @@ internal static void InitPrefabs()
                     .sharedMaterial;
             }
 
-            if (_scrollPrefab == null) _scrollPrefab = SearchUtilities.Find("BrittleHollow_Body/Sector_BH/Sector_NorthHemisphere/Sector_NorthPole/Sector_HangingCity/Sector_HangingCity_District2/Interactables_HangingCity_District2/Prefab_NOM_Scroll").InstantiateInactive().Rename("Prefab_NOM_Scroll").DontDestroyOnLoad();
+            if (_scrollPrefab == null)
+            {
+                _scrollPrefab = SearchUtilities.Find("BrittleHollow_Body/Sector_BH/Sector_NorthHemisphere/Sector_NorthPole/Sector_HangingCity/Sector_HangingCity_District2/Interactables_HangingCity_District2/Prefab_NOM_Scroll").InstantiateInactive().Rename("Prefab_NOM_Scroll").DontDestroyOnLoad();
+            }
 
             if (_computerPrefab == null)
             {
                 _computerPrefab = SearchUtilities.Find("VolcanicMoon_Body/Sector_VM/Interactables_VM/Prefab_NOM_Computer").InstantiateInactive().Rename("Prefab_NOM_Computer").DontDestroyOnLoad();
-                _computerPrefab.transform.rotation = Quaternion.identity;
```

Comment:
No longer need to zero-out the prefabs manually since GeneralPropBuilder (and by extension DetailBuilder for the types that use it) always sets the object rotation to Quaternion.identity if no rotation is provided.